owa/app/js/components/bodyLayout.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { TabContent, TabPane, Nav, NavItem, NavLink, 
+        Card, Button, CardTitle, CardText, Row, Col, Form, FormGroup, 
+        Label, Input, FormText } from 'reactstrap';
+import classnames from 'classnames';
+
+
+export default class BodyLayout extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      activeTab: '1'
+    };
+  }
+
+  toggle(tab) {
+    if (this.state.activeTab !== tab) {
+      this.setState({
+        activeTab: tab
+      });
+    }
+  }
+  render() {
+    return (
+      <div>
+        <Nav tabs>
+          <NavItem>
+            <NavLink
+              className={classnames({ active: this.state.activeTab === '1' })}
+              onClick={() => { this.toggle('1'); }}
+            >
```

Comment:
These have nothing with the setup of the module.
The setup should in such a way that if we decide not to put the three user interfaces of log entries, identifier sources, etc, we do not have to remove any of the files we committed. Putting it in a different way, the setup would be like adding any hello world autogenerated owa module.
Adding the navigation bars would be a separate ticket of its own.

Reply:
okay cancelling the PR and doing another one right away